src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from '../components/PersonLink/PersonLink';
+import { Loader } from '../components/Loader/Loader';
+
+function findPersonByName(name: string | null, people: Person[]) {
+  return people.find(person => person.name === name);
+}
+
+export const PeoplePage = () => {
+  const { personSlug } = useParams();
+  const [people, setPeople] = useState<Person[]>();
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setErrorMessage('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="container">
+        <div className="block">
+          <div className="box table-container">
+            {isLoading ? (
+              <Loader />
+            ) : (
+              <>
+                {errorMessage !== '' && (
+                  <p data-cy="peopleLoadingError" className="has-text-danger">
+                    {errorMessage}
+                  </p>
+                )}
+
+                {people?.length === 0 ? (
+                  <p data-cy="noPeopleMessage">
+                    There are no people on the server
+                  </p>
+                ) : (
```

Comment:
I think it is better to do it not through the ternary operator, but simply through if

Reply:
Yeah, you right. PeoplePage need refactoring - too hard to read right now